### ISSUE TITLE
set-output command  deprecated

### ISSUE THREAD

**Issue by cech12** [mikepenz/gradle-dependency-submission#153]
When I run your action, I get the following warning:

The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

detailed log: https://pastebin.com/m0ahAQg8

**Comment by cech12** [comment]
@mikepenz 
Unfortunately, this issue was not fixed by the 0.8.1 release :/

```
Run mikepenz/gradle-dependency-submission@v0.8.1
📘 Reading input values
  No 'gradle-build-module' passed, using ':'
🔨 Processing gradle dependencies for root module - ':'
  Completed retrieving the 'dependencies' for configuration 'compileClasspath' within 9385ms
📄 Parsing gradle dependencies graph - ':'
  Dependency output of 2[18](https://github.com/cech12/BucketLib/actions/runs/3338939803/jobs/5527276926#step:6:21) lines
  Completed parsing 189 dependency associations within 2ms
📦️ Preparing Dependency Snapshot - ':'
  Connection 3 direct dependencies
  Connection 90 indirect dependencies

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Notice: Submitting snapshot...
```

**Comment by mikepenz** [comment]
Opened a PR here: https://github.com/github/dependency-submission-toolkit/pull/34
